apis/python/src/tiledbsoma/dataframe.py

```diff
@@ -100,10 +100,15 @@ def _create_empty(
                 else:
                     raise TypeError(f"Unsupported dtype {dtype}")
 
+            # Default 2048 mods to 0 for 8-bit types and 0 is an invalid extent
+            extent = create_options.dim_tile(index_column_name)
```

Comment:
@bkmartinjr no because the two other callsites both have hard-coded `dtype` of `np.int64`:

* https://github.com/single-cell-data/TileDB-SOMA/blob/main/apis/python/src/tiledbsoma/sparse_nd_array.py#L82
* https://github.com/single-cell-data/TileDB-SOMA/blob/main/apis/python/src/tiledbsoma/dense_nd_array.py#L82